src/main/java/de/zalando/aruha/nakadi/partitioning/InvalidOrderingKeyFieldsException.java

```diff
@@ -0,0 +1,17 @@
+package de.zalando.aruha.nakadi.partitioning;
```

Comment:
That wasn't by intention. Usually I like to structure my packages by business logic not technically, that's probably why this class ended up here. But I see, we have a technical structure in the project, I'll move it. Thanks for pointing out!